tldw_Server_API/app/core/AuthNZ/repos/rate_limits_repo.py

```
@@ -25,6 +25,58 @@ def _is_postgres_backend(self) -> bool:
         """Return True when the underlying DatabasePool is using PostgreSQL."""
         return bool(getattr(self.db_pool, "pool", None))
 
+    async def _ensure_postgres_account_lockouts_schema(self, conn: Any) -> None:
```

Ruff (0.15.9)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `conn`

(ANN401)